CONTRIBUTING.rst

```diff
@@ -52,22 +52,26 @@ Have questions or need help?
 Development Environment
 =======================
 
-* To run the complete test suite you will need `ZFS`_ and `Docker`_ installed.
-  The recommended way to get an environment with these installed is to use the included ``Vagrantfile`` which will create a pre-configured CentOS 7 virtual machine.
-  Vagrant 1.6.2 or later is required.
-  Once you have Vagrant installed (see the `Vagrant documentation <https://docs.vagrantup.com/v2/>`_) you can run the following to get going:
+To run the complete test suite you will need `ZFS`_ and `Docker`_ installed.
+The recommended way to get an environment with these installed is to use the included ``Vagrantfile`` which will create a pre-configured CentOS 7 virtual machine.
```

Comment:
This was an issue separate to any changes in this branch, but it would be good to link to the `Vagrantfile` download here, otherwise "the included Vagrantfile" doesn't make much sense and might leave me wondering "included where?" - also need to differentiate between our tutorial and dev Vagrant boxes, or someone might reference the wrong `Vagrantfile`.